lib/project-view.coffee

```diff
@@ -37,6 +45,7 @@ module.exports = ProjectView =
         @subscribeUpdateEvents()
         # Initally update the root names
         @updateRoots(@treeView.roots)
```

Comment:
Should be

```
          @regex = new RegExp(atom.config.get('project-view.regexMatch'))
          @updateRoots(@treeView.roots)
```

or? Otherwise `@regex` will be undefined unless `updateRoots()` you update the `regexMatch` string via the settings pane.